pennylane/fermi/fermionic.py

```diff
@@ -324,6 +324,105 @@ def to_mat(self, n_orbitals=None, format="dense", buffer_size=None):
             wire_order=list(range(largest_order)), format=format, buffer_size=buffer_size
         )
 
+    def shift_operator(self, initial_position, final_position):
+        r"""Shifts an operator in the FermiWord from ``initial_position`` to ``final_position`` by applying the fermionic anti-commutation relations.
+
+    There are four `anti-commutator relations <https://en.wikipedia.org/wiki/Creation_and_annihilation_operators#Creation_and_annihilation_operators_in_quantum_field_theories>`_ for fermionic creation, :math:`a`, and annihilation :math:`a^{\dagger}` operators:
+
+    .. math::
+        \begin{align*}
+            \left\{ a_i, a_j \right\} &= 0,\\\\
+            \left\{ a^{\dagger}_i, a^{\dagger}_j \right\} &= 0,\\\\
+            \left\{ a_i, a^{\dagger}_j \right\} &= \delta_{ij},\\\\
```

Comment:
It might be better to put all these three in a single line (instead of newline for each) and use `\quad` to add the required separation.